cadc-vos-server/src/main/java/ca/nrc/cadc/vos/server/transfers/TransferRunner.java

```diff
@@ -387,16 +388,19 @@ private void doTransferRedirect(Transfer trans, List<Parameter> additionalParame
             // standard redirect
             StringBuilder sb = new StringBuilder();
             sb.append("/").append(job.getID()).append("/results/transferDetails");
+
             try {
                 AuthMethod authMethod = AuthenticationUtil.getAuthMethod(AuthenticationUtil.getCurrentSubject());
-                URL serviceURL = regClient.getServiceURL(serviceURI, Standards.VOSPACE_TRANSFERS_20, authMethod);
+                URL serviceURL = regClient.getServiceURL(serviceURI,  Standards.VOSPACE_TRANSFERS_20, authMethod);
                 URL location = new URL(serviceURL.toExternalForm() + sb.toString());
                 String loc = location.toExternalForm();
                 log.debug("Location: " + loc);
                 syncOutput.setHeader("Location", loc);
                 syncOutput.setResponseCode(HttpURLConnection.HTTP_SEE_OTHER);
                 return;
-            } catch (MalformedURLException bug) {
+            } catch (MalformedURLException | UnsupportedOperationException bug) {
```

Comment:
What throws UnsupportedOperationException here?  It's not a bug so needs to be handled differently than the MalformedURLException.

Reply:
That was left over from the previous iteration of the function (where /pkg endpoints where generated here...) - removed that exception.